src/utils/util.js

```diff
@@ -0,0 +1,6 @@
+const REGEXP_DOMAIN = /(?:[^/]*\/){3}/;
+
+export function getDomain(url) {
+  const matched = url.match(REGEXP_DOMAIN);
```

Comment:
Just curious, why not using Node.js URL module?

And I expected the return value of this function is `example.com` when url is `https://example.com/test`, but it seems that the actual return value is `https://example.com/`. Would you declare more right name? (It's called `origin` in Node.js URL module https://nodejs.org/dist/latest/docs/api/url.html)